sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/validation/CapabilityValidator.java

```diff
@@ -68,6 +68,27 @@ private void validate(PodSpec podSpec) throws CapabilityValidationException {
                 }
             }
         }
+
+        if (!podSpec.getSecrets().isEmpty()) {
+            String fileBAsedSecretSupportMessage = "This cluster's DC/OS version does not support file-based Secrets";
+            // TODO(MB) : Change validator if we decide to support DCOS_DIRECTIVE label
+            // String envBasedSecretSupportMessage =
+            //        "This cluster's DC/OS version does not support environment-based Secrets";
+            String secretsSupportMessage =
+                    "Dcos-commons (SDK) does not support Secrets for this cluster's DC/OS version";
```

Comment:
`This service does not support...` (the reader of this error may not know about dcos-commons or SDK)